src/features/user/TreeMapper/index.tsx

```diff
@@ -40,22 +40,19 @@ function TreeMapper(): ReactElement {
     PlantLocationSingle | PlantLocationMulti | null
   >(null);
   const [links, setLinks] = React.useState<Links>();
-  const { tenantConfig } = useTenant();
   const { redirect, setErrors } = React.useContext(ErrorHandlingContext);
   async function fetchTreemapperData(next = false) {
     setIsDataLoading(true);
     setProgress(70);
 
     if (next && links?.next) {
       try {
-        const response =
-          await getAuthenticatedRequest<ExtendedScopePlantLocations>({
-            tenant: tenantConfig?.id,
-            url: links.next,
-            token,
-            logoutUser,
+        const response = await getApiAuthenticated<ExtendedScopePlantLocations>(
+          links.next,
+          {
```

Comment:
Add a comment here to indicate that the url contains query parameters, and is being passed as is since no additional parameters are being added.